docs/src/quickstart/cythonize.rst

```diff
@@ -84,7 +111,8 @@ typed values directly) and from Python (wrapping values in Python
 objects). In fact, ``cpdef`` does not just provide a Python wrapper, it also
 installs logic to allow the method to be overridden by python methods, even
 when called from within cython. This does add a tiny overhead compared to ``cdef``
-methods.
+methods. Again, Cython provides ``@ccall`` decorator which provides same
```

Comment:
```suggestion
methods. Also Cython provides ``@ccall`` decorator which provides same
```

Or at least not "again"

Reply:
What's wrong with "again" here? It was mentioned before.

Reply:
It's first time mention of ccall in this document, again should refer to something we saw/read